# ESM-ARG

Antibiotic resistance genes (ARGs) are critical for bacterial resistance, challenging global health and the environment, affecting millions worldwide. Predicting if a given gene sequence is ARG is vital for combating resistance spread and treatment of bacterial infections. Traditional ARG prediction relies on alignment-based sequence similarities with known ARGs. However, the alignment-based approach is limited by its dependence on sequence similarity and choice of reference databases, which may miss novel ARGs and potentially result in high false negative rates. To address this, we developed ESM-ARG, a machine learning-based framework for classifying ARGs, which incorporates a pretrained Evolutionary Scale Modeling (ESM-2), trained on a vast protein dataset for generating protein sequence embeddings and uses machine learning (ML) models to identify and classify ARGs. ESM-ARG takes raw protein sequences as input, generates protein sequence embeddings from the pretrained model, and then uses ML models to predict whether the input protein sequence is an ARG or not. If the input protein sequence is an ARG, another multiclass classification model is used to identify the antibiotic class to which the input protein is resistant. To train ESM-ARG, we used a dataset curated from 10 available databases containing ARGs from 28 antibiotic resistance categories. Our extensive comparative experimental analysis, including 5-fold cross-validation results, demonstrates that ESM-ARG outperforms different state-of-the-art methodologies by achieving high accuracy (> 0.98), precision (> 0.99), recall (> 0.98), F1-score (> 0.98), and higher Matthewâ€™s correlation coefficients (MCCs) of (> 0.97). Moreover, the effectiveness and robustness of ESM-ARG provide valuable insights for combating antibiotic resistance.
